### ISSUE TITLE
Add password recovery

### ISSUE THREAD

**Issue by neuhaus** [loomchild/sojourner-web#16]
There is no way to recover a lost password at the moment

**Comment by loomchild** [comment]
You are absolutely right, it's on a TODO list. The only way to do it is to send me an email and I could manually send a reset password message to the email address of the account owner. But yeah, this needs to be added for the next year.
